app-config-utils/src/__snapshots__/index.test.ts.snap

```diff
@@ -13,6 +13,25 @@ exports[`generateModuleText creates config module 1`] = `
     "
 `;
 
+exports[`generateModuleText creates config module with doNotLoadConfig 1`] = `
+"
+      const globalNamespace = (typeof window === 'undefined' ? globalThis : window) || {};
+
+      const config = globalNamespace._appConfig;
+
+      if (typeof config === 'undefined') {
+        throw new Error('Config is not loaded in _appConfig');
+      }
+
+      export { config };
+      export default config;
+    
+      export function currentEnvironment() {
+        return globalNamespace._appConfigEnvironment || \\"test\\";
```

Comment:
Side question - I see that `@app-config/inject` sets `window._appConfigEnvironment`, which is great so that we can still set this at runtime. But where does `@app-config/inject` get this from? `ENV`/`NODE_ENV`/`APP_CONFIG_ENV`/etc?

Reply:
Exact same semantics as normal app-config, yup (it uses the `environment` output of `loadConfig`)